content-src/experiments/email-tabs.yaml

```diff
@@ -0,0 +1,75 @@
+id: 20
+title: 'Email Tabs'
+slug: email-tabs
+is_featured: false
```

Comment:
Should this be `true`?
I see you de-featured Lockbox below, but wasn't sure if there was any other featured e'speriments.

Reply:
Similarly (but unrelatedly), not sure if you want an `is_dev: true` for some foot-gun protection, and the ability to deploy this to dev without the risk of it riding the train to production [yet].